### PR TITLE
Db models

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: packages/wallet-core/db/schemas/1-init.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "name": "wallet",
+      "primaryKey": "id",
+      "properties": {
+        "id": {
+          "type": "int"
+        },
+        "name": {
+          "type": "string"
+        },
+        "publicKey": {
```
Comment (sk91) [inline packages/wallet-core/db/schemas/1-init.js]: publicKey should probably be 'address'

File: packages/wallet-core/db/schemas/1-init.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          "type": "string"
+        },
+        "privateKey": {
```
Comment (sk91) [inline packages/wallet-core/db/schemas/1-init.js]: Are we going to save private keys to db?

Comment (maycon-mello) [reply packages/wallet-core/db/schemas/1-init.js]: I think so for this first bets. We could remove it after getting the backup wallet feature ready in next bets.  Also, the idea is to enable encryption in the db to improve security

File: packages/wallet-core/db/schemas/1-init.js (hunk 1 above)
Comment (maycon-mello) [reply packages/wallet-core/db/schemas/1-init.js]: Sure, 'address' is better. I will change this name.

I tried to replicate the same model from the desktop wallet, that's why it was created this way.

https://github.com/SelfKeyFoundation/Identity-Wallet/blob/master/src/main/wallet/wallet.js#L24

Comment (sk91) [reply packages/wallet-core/db/schemas/1-init.js]: yes, it is wrong there, will be changed when we'll have time for it

**Pull request merged**